src/components/widgets/ColorscalePicker.js

```diff
@@ -1,41 +1,95 @@
-import ColorscalePicker, {Colorscale} from 'react-colorscales';
+import ColorscalePicker, {
+  Colorscale,
+  COLOR_PICKER_CONSTANTS,
+} from 'react-colorscales';
+import Dropdown from './Dropdown';
+import Info from '../fields/Info';
 import PropTypes from 'prop-types';
-import React, {Component} from 'react';
+import React, {Component, Fragment} from 'react';
 
 class Scale extends Component {
-  constructor(props) {
-    super(props);
+  constructor() {
+    super();
+
     this.state = {
+      selectedColorscaleType: 'sequential',
```

Comment:
note: in a group of traces, this default makes less sense, and should be overrideable to `categorical`

Reply:
yup, will look at how to select the initial state better in a subsequent pr..
but the structure is there to add it easily now :)